tests/migration/migration.go

```diff
@@ -2879,8 +2879,12 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 
 			for _, line := range cpuLines {
 				lineSplits := strings.Fields(line)
+				// Range will be found when there are disabled/not plugged cpus
+				if strings.Contains(lineSplits[1], "-") {
+					continue
+				}
 				cpu, err := strconv.Atoi(lineSplits[1])
-				Expect(err).ToNot(HaveOccurred(), "cpu id is non string in vcpupin output")
+				Expect(err).ToNot(HaveOccurred(), "cpu id is non string in vcpupin output", vcpuPinOutput, "\n", cpuLines)
```

Comment:
Aren't you supposed to use the printf format here?

Reply:
I can but it is not required. Actually I think the `vcpuPinOutput` is only needed part. Pushed the change

Reply:
Interesting, do you know what the actual logic is? Like, if there's a `%` in the first string then it's used as a format string, otherwise all the strings are concatenated? Just curious

Reply:
```
return fmt.Sprintf(optionalDescription[0].(string), optionalDescription[1:]...) + "\n"
```
They do.

Reply:
@jean-edouard 
Are you good with
```
[FAILED] cpu id is non string in vcpupin output%!(EXTRA string= VCPU   CPU Affinity
  ----------------------
   0      2
   1      3
   2      0-7
   3      0-7
   4      0-7
   5      0-7
   6      0-7
   7      0-7

  )
```
? I can add the %s if you want me to remove the `%!(EXTRA`...

Reply:
Ok, so... You need the first string to contain formatting directives, no? Like https://go.dev/play/p/MKhwofhfJzj

Reply:
> @jean-edouard Are you good with [...]

Just saw that, so basically if the format string is malformed it still prints the stuff, nice :)
Sounds good then.